contracts/src/v0.8/mocks/VRFCoordinatorV2Mock.sol

```diff
@@ -37,6 +42,8 @@ contract VRFCoordinatorV2Mock is VRFCoordinatorV2Interface {
     uint96 balance;
   }
   mapping(uint64 => Subscription) s_subscriptions; /* subId */ /* subscription */
+  mapping(address => mapping(uint64 => bool)) s_consumer_added; /* consumer */ /* subId */ /* added */
```

Comment:
Why do we need both s_consumer_added and s_consumers? Can it be simplified to just the latter?

also naming nit: s_consumer_added -> s_consumerAdded

Reply:
I'll take care of the naming nit

Reply:
Since this is just a mock contract, gas usage doesn't matter. I kinda prefer iterating through the array just to keep this contract as simple as possible

Reply:
I've removed the mapping variable and now just rely on the s_consumers array.